yellowbrick/text/postag.py

```diff
@@ -431,17 +522,16 @@ def show(self, outpath=None, **kwargs):
 
 
 def postag(
-    X,
-    y=None,
-    ax=None,
-    tagset="penn_treebank",
-    colormap=None,
-    colors=None,
-    frequency=False,
-    stack=False,
-    **kwargs
+        X,
+        y=None,
+        ax=None,
+        tagset="penn_treebank",
+        colormap=None,
+        colors=None,
+        frequency=False,
+        stack=False,
+        **kwargs
```

Comment:
Please make sure to add the extra arguments to the quick method, and make sure that they're passed to the visualizer.

Reply:
Ah thanks I missed this. Resolved in d802d1f